app/models/form_profile.rb

```diff
@@ -102,7 +102,7 @@ class FormProfile
     coe: ['26-1880'],
     adapted_housing: ['26-4555'],
     intent_to_file: ['21-0966'],
-    ivc_champva: %w[10-7959F-1 10-7959C],
+    ivc_champva: %w['10-7959C'],
```

Comment:
You have to remove the `%w` thats only for Arrays with multiple strings.

Reply:
Ahhhh my bad!